ocaml/runtime/fiber.c

```diff
@@ -554,6 +570,20 @@ int caml_try_realloc_stack(asize_t required_space)
         link->sp = (void*)((char*)Stack_high(new_stack) -
                            ((char*)Stack_high(old_stack) - (char*)link->sp));
       }
+      if (link->async_exn_handler > (char*) Stack_base(old_stack)
+          && link->async_exn_handler <= (char*) Stack_high(old_stack)) {
```

Comment:
Fixed, but I took this from line 430.  Is that wrong?

Reply:
I suspect so! On closer inspection I don't think it can matter (the exn handler can never be the last thing pushed during a try_realloc_stack, so < vs <= on base makes no difference)